clang/lib/CodeGen/TargetBuiltins/ARM.cpp

```diff
@@ -5500,8 +5502,13 @@ Value *CodeGenFunction::EmitAArch64BuiltinExpr(unsigned BuiltinID,
 
     llvm::Function *F = CGM.getIntrinsic(Intrinsic::write_register, Types);
     llvm::Value *ArgValue = EmitScalarExpr(E->getArg(1));
-
-    return Builder.CreateCall(F, { Metadata, ArgValue });
+    llvm::Value *Result = Builder.CreateCall(F, {Metadata, ArgValue});
```

Comment:
Move this into an `else` block?

Reply:
Oh, my bad, I had the DAG stuff on my mind so I thought that the `CreateCall` would be wasted. The current code is fine.